addons/website_slides/models/slide_slide.py

```diff
@@ -569,26 +570,28 @@ def _action_vote(self, upvote=True):
         channel = slide_id.channel_id
         karma_to_add = 0
 
+        reason_lesson_upvoted = _('Lesson upvoted')
```

Comment:
this makes me tell that reason should also be a selection field (if there are only few and fixed reasons)

Reply:
The reason field is also used when a manual entry is done (only available for high level users) and the goal is to provide the user the freedom to explain any reason. So I wouldn't use a selection field here. I understand your point if it was only used by automated process with a few and fixed reason indeed but it is not the goal here.